main.tf

```diff
@@ -477,3 +477,56 @@ resource "aws_cloudwatch_log_stream" "cloudwatch_stream" {
   name           = "${var.app_name}-log-stream"
   log_group_name = aws_cloudwatch_log_group.cloudwatch_group.name
 }
+
+resource "aws_autoscaling_schedule" "ecs_non_prod_scale_down" {
+  count                  = local.is-production == true ? 1 : 0
```

Comment:
this is going to create an ASG if the tag `is-production` is true and therefore it will create an ASG for prod accounts; 

However we want an ASG for non-prod accounts